### PR TITLE
returnKeyType is not working in TextView

### PR DESCRIPTION
**Pull request by b02505048**

### CODE REVIEW

**Comment by ikoevska** [comment]
Just checked it on an iPhone and it works.

Here's the sample code  tested it with:
```HTML
<TextView editable="true" returnKeyType="search">
       <FormattedString>
             <Span text="You can use text attributes such as " />
             <Span text="bold, " fontWeight="Bold" />
             <Span text="italic " fontStyle="Italic" />
             <Span text="and " />
             <Span text="underline." textDecoration="Underline" />
        </FormattedString>
</TextView>
```

What's the scenario that didn't work for you?

**Comment by b02505048** [comment]
Yeah you're right, it only works on iPhone. 
For Android, it's still not working at all so far.

File: content/docs/en/elements/components/text-view.md [hunk 1]
```diff
@@ -47,7 +47,6 @@ To apply multiple styles to the text in your `<TextView>`, you can use `<Formatt
 | `editable` | `Boolean` | When `true`, indicates that the user can edit the contents of the container.
 | `maxLength` | `Number` | Sets the maximum number of characters that can be entered in the container.
 | `keyboardType` | `KeyboardType` | Shows a custom keyboard for easier text input.<br/>Valid values: `datetime`, `phone`, `number`, `url`, or `email`.
-| `returnKeyType` | Gets or sets the label of the return key.<br/>Valid values: `done`, `next`, `go`, `search`, or `send`.
```
Comment (ikoevska) [inline content/docs/en/elements/components/text-view.md]: Maybe retain the line but say:

```
| `returnKeyType` | (iOS-only) Gets or sets the label of the return key.<br/>Valid values: `done`, `next`, `go`, `search`, or `send`.
```

or

```
| `returnKeyType` | Gets or sets the label of the return key. Currently supported only on iOS.<br/>Valid values: `done`, `next`, `go`, `search`, or `send`.
```

Comment (b02505048) [reply content/docs/en/elements/components/text-view.md]: Oh yeah, that's way better!

**Pull request merged**